Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14149,6 +14225,21 @@ end
 					},
 					Notes = "Returns the item that has been read from the specified INI file value. If the value is not present in the INI file, the DefaultValue is stored in the file and parsed as the result. Returns empty item if the value cannot be parsed. ",
 				},
+				GetSnowStartHeight =
+				{
+					Params =
+					{
+						{
+							Name = "Biome",
+							Type = "EMCSBiome",
+						},
+					},
+					Returns =
+					{
+						{ Type = "number" },
+					},
+					Notes = "Returns the height at which snow will start falling in the {{EMCSBiome}|Biome}}.",
```

Comment:
What does it return for hot biomes and very-cold biomes? Perhaps add a link to `IsBiomeCold` et al.